airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -285,12 +280,12 @@ def _fail_on_errors(self, errors: List[str]):
         if len(errors) > 0:
             pytest.fail("\n".join(errors))
 
-    def test_property_type_is_not_array(self, connector_specification: dict):
+    def test_property_type_is_not_array(self, connector_spec: dict):
```

Comment:
@flash1293 do you have context on why you needed a new fixture? Replacing `connector_specification` with `connector_spec` seems to fix the problem, but I might be missing something here